src/utils/fetchWithTimeout.ts

```diff
@@ -0,0 +1,15 @@
+const fetchWithTimeout = (url, options = {}) => {
```

Comment:
`options` can be typed as RequestInit perhaps? Then `// @ts-ignore` can be removed

Reply:
Done